src/service/resources-impl.js

```diff
@@ -193,8 +193,17 @@ export class ResourcesImpl {
       this.ampdoc.getVisibilityState()
     );
 
-    /** @private {?IntersectionObserver} */
-    this.intersectionObserver_ = null;
+    /**
```

Comment:
The bigger question I have: if this continues to diverge in major ways, would it be easier to just fork this whole manager for the InOb experiment?

Reply:
Good question. I only have a few small changes left to do, so this is nearly the final state.

Also I'm concerned about bundle size since we can't leverage module/nomodule or other tricks easily.